Node JS/session fswd 01/UI/food-adda/src/pages/Login.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import { Form, Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { toast } from "react-toastify";
+import { Signup } from "../components/Signup";
+
+export const Login = () => {
+  // State variables
+  const [modalShow, setModalShow] = useState(false);
+
+  // Login handler function
+  const loginHandler = (e) => {
+    e.preventDefault();
+
+    const email = e.target.elements.formBasicEmail.value;
+    const password = e.target.elements.formBasicPassword.value;
+    const role = e.target.elements.group1.value;
+    console.log(email, password, role);
+    // Toast success message
+    toast.success("Login Successful", {
```

Comment:
Make a utility function which would have all of these options like autoCloser and all, after that all you have to do is call that function with custom message and probably with type(success/error)